lib/alex_koin/slack.ex

```diff
@@ -57,10 +59,22 @@ defmodule AlexKoin.SlackRtm do
   end
 
   defp create_reply(user, message, {:balance, _text}, _slack) do
+    Logger.info "#{user.first_name} is asking about their balance.", ansi_color: :green
     balance = SlackCommands.get_balance(user_wallet(user))
-
     {"You have #{balance}:akc:.", message_ts(message)}
   end
+  defp create_reply(user, message, {:other_balance, text}, slack) do
+    regex = ~r/<@(?<slack_id>[A-Z0-9]+)>/
+
+    if Regex.match?(regex, text) do
+      %{ "slack_id" => slack_id } = Regex.named_captures(regex, text)
+      user_to_check = SlackCommands.get_or_create(slack_id, slack)
+      balance = SlackCommands.get_balance(user_wallet(user_to_check))
```

Comment:
Use pipes for at the least, get_balance and user_wallet.

Would be nice to have the others use pipes, but I also realize they are used later for logging and the returned message. I'm ok with that.